public/index.php

```diff
@@ -1,55 +0,0 @@
-<?php
```

Comment:
I guess this is still used by dusk. Maybe it can be updated to use octane as well. And maybe do something about `artisan serve`